packages/neuron-ui/src/components/TransactionType/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
```

Comment:
Unnecessary import

Reply:
It will throw an error without React when using Jsx by `precommit lint`.

Reply:
We may check later why `import React` is still required because it can be omitted since React@17

Ref: 
- use JSX without importing React: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-a-jsx-transform